dashboard/views.py

```diff
@@ -227,6 +227,13 @@ def get(self, request):
 
             qa_qs = QAUpdates.objects.filter(cap_id=capinstance).values()
             qalist = list(qa_qs)
+            
+            for qa in qalist:
+                qa['name'] = qa['name'][2:-2].replace("', '","\n").replace("'", "")
+                qa['title'] = qa['title'][2:-2].replace("', '","\n").replace("'","")
+                qa['catalog_url'] = qa['catalog_url'][2:-2].replace("', '","\n").replace("'","")
+                qa['organization'] = qa['organization'][2:-2].replace("', '","\n").replace("'","")
+                qa['metadata_type'] = qa['metadata_type'][2:-2].replace("', '","\n").replace("'","")
```

Comment:
not sure what's going on here, can i see an example?

Reply:
Reformatting items, so they display like this:
<img width="567" alt="Screen Shot 2021-12-10 at 12 50 39 PM" src="https://user-images.githubusercontent.com/57103986/145626404-0f0ac18f-e440-488c-b708-cd84ed51b57f.png">

instead of this:
<img width="803" alt="Screen Shot 2021-12-10 at 12 49 51 PM" src="https://user-images.githubusercontent.com/57103986/145626498-334f8241-4f94-4760-b4ce-f0b024b64552.png">

Reply:
The value contained in `qa['name']` is a json string. Json strings can be parsed using `json` python package. `json.loads(string_value)` will give a dictionary representation of the json string. I think this can be done better using variables and additional functions.

```
def generate_string(json_string):
    """
    Generate the desired string from a json string
    :param json_string:
    :return: string
    """
    ret_str = ""
    json_dict = json.loads(json_string)
    # returning the key values that were loaded and joining them with a new line
    return "\n".join([f"{key}: {value}" for key, value in json_dict.items()])
    

keys = ['name', 'title', 'catalog_url', 'organization', 'metadata_type']
for qa in qalist:
    for key in keys:
        if key in qa:
            # either this
            qa[key] = generate_string(qa[key])

            # or this
            qa[key] = qa['name'][2:-2].replace("', '","\n").replace("'", "")
```

However, having a json string in the name column is not ideal. If there is a way to avoid this that would be better. Maybe during the data fetching and/or during the database population point. That being said, if it requires a lot of code changes, I suggest against it.

Reply:
Created json_converter method to handle this